src/OpenConext/Profile/Value/Arp.php

```diff
@@ -186,4 +187,40 @@ public function getMotivationFor(
         }
         return '';
     }
+
+    /**
+     * Accordingly to https://www.pivotaltracker.com/n/projects/1453004/stories/187607790
+     * the ARP data should only contain the following keys: value, source, motivation.
+     * This function will filter out all other keys, which are to be ignored.
+     */
+    private static function sanitizeARPData(array $arp): array
```

Comment:
Nitpick: sanitizeArpData. Abbreviations are considered normal words.

Reply:
The calling part must also be updated now ;) 

TIL: calling methods is not case sensitive :thinking:

Reply:
Apparantly (didn't knew that either). Will fix though